src/components/reusable/table/table-header.scss

```diff
@@ -0,0 +1,81 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: table-cell;
+  vertical-align: middle;
+  @include typography.type-body-02;
```

Comment:
The design has a smaller font, looks like UI-03

Reply:
The font is the one trying to match dropdown's, I checked and it's type-body-02

Reply:
That doesn't seem right. Checking the Figma, the header cell text is definitely 12px.

Reply:
Oops you're right, that's the pagination and this is the header.